src/picasso-def/components/treemap.js

```diff
@@ -70,13 +76,23 @@ export const treemap = () => ({
       .paddingTop((node) => {
         // only want headers at level 1
         if (node.depth === 1 && node.height > 1) {
+          const nodeWidth = node.x1 - node.x0;
           const nodeHeight = node.y1 - node.y0;
-          const showLable = nodeHeight > TREEMAP_DEFINES.HEADER_HEIGHT * 2;
+          const nodeArea = nodeWidth * nodeHeight;
+          const showLable = nodeArea > MIN_AREA_FOR_HEADERS;
           node.header = true;
           node.showLable = showLable;
-          return (labels.headers || labels.auto) && showLable ? TREEMAP_DEFINES.HEADER_HEIGHT : 1;
+          if (labels.headers || labels.auto) {
+            if (smallLayoutMode) {
+              return 0;
+            } else if (showLable && !smallLayoutMode) {
+              return TREEMAP_DEFINES.HEADER_HEIGHT;
+            } else if (!showLable) {
+              return TREEMAP_DEFINES.COLLAPSED_HEADER_HEIGHT;
+            }
+          }
         }
-        return 1;
+        return 0;
```

Comment:
why?
this change makes the horzinontal and vertical padding between rects different

Reply:
This was in between changes, I did for some experimenting and missed to change back. Good catch ! I will revert it !

Reply:
did you also check the other return 0; (for smallLayoutMode)?